### PR TITLE
Add Solr config update tip, fixes #15

### LINKED ISSUE #15: search_api_solr shows "Outdated solr configuration"
Today I tried to setup a Solr server.
After following the instructions and clicking save, I recieved 2 messages:

- "The server was successfully saved."
- "You are using outdated Solr configuration set. Please follow the instructions described in the [README.md](https://solr-d10-demo.ddev.site/modules/contrib/search_api_solr/README.md) file for setting up Solr."

outdated_config

It states it can connect OK is the view table in Drupal 10. And the configured version and detected version appear to match:

|  |  |
| --- | --- |
| Server Connection | The Solr server could be reached. |
| Core Connection | The Solr core could be accessed (latency: 10.037945343018 ms).|
|Configured Solr Version |8.11.2 |
|Detected Solr Version |8.11.2 |

However, I am unfamiliar with Solr so not really sure if there are any "issues". 
We may need to update the configuration or, perhaps, add a note on the addon page to "ignore the warning".

## Test

1.. Follow installation guide.
2. Click "save"
3. Check messages.

DDEV: `1.24.1`
Drupal: `10.0.2`

### PR DESCRIPTION
**Pull request by gitressa**
## The Issue
Fixes 
* #15.

## How This PR Solves The Issue

Adds documentation on how to fix outdated Solr config files message.

Also updates the "Alternate Core Name" section, the first step (`1. Remove the #ddev-generated at the top of the file.`) isn't necessary, it seems?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,10 +57,8 @@ services:
     environment:
     - SOLR_CORENAME=somecorename
 ```
-
-1. Remove the #ddev-generated at the top of the file.
```
Comment (rfay) [inline README.md:53]: Why did you remove the first item here?

**Review by rfay — CHANGES_REQUESTED** [review]
It all looks great, just the one question about why you removed #ddev-generated notice.

**Comment by rfay** [comment]
If you don't remove the `#ddev-generated` then the file will be recreated with the default file on `ddev restart`.

**Comment by gitressa** [comment]
I removed it because after creating the file _.ddev/docker-compose.solr-env.yaml_, adding this:

```
services:
  solr:
    environment:
    - SOLR_CORENAME=somecorename
```

and restarting DDEV, at no point do I see a line saying `#ddev-generated`, so I removed that first line ...

Or is that first line about something else?

File: README.md (hunk 1 above)
Comment (rfay) [reply README.md:53]: Ah, I didn't see the context `If you want to use a core name other than the default "dev", add a .ddev/docker-compose.solr-env.yaml with these contents, using the core name you want to use:`, thought it was referring to docker-compose.solr.yaml.

**Review by rfay — APPROVED** [review]

**Comment by gitressa** [comment]
Great, thanks!

**Comment by rfay** [comment]
Thanks for the contribution and for keeping your eye on this repo. You know and use solr far more than I do, and it needs knowledgable folks following it.

**Comment by gitressa** [comment]
You're welcome, I am glad I could help. It's a privilege to assist you with DDEV, it's an amazing web developer tool.

**Pull request merged**